backend/api/src/api/v1/gql/subscription/user.rs

```diff
@@ -36,7 +35,7 @@ impl UserSubscription {
 
         let mut subscription = global
             .subscription_manager
-            .subscribe(format!("user.{}.display_name", user_id))
+            .subscribe(format!("user:{}:display_name", user_id))
```

Comment:
Why did you switch from using `.` to `:`?
Redis suggests to use dots: https://redis.io/docs/manual/pubsub/#pattern-matching-subscriptions
It probably doesn't matter though.

Reply:
this isnt the pattern based subscription its just a regular one, and in redis namespaces are generally denoted by colons `:`
https://redis.com/blog/5-key-takeaways-for-developing-with-redis/